src/app/model/Types.ts

```diff
@@ -10,3 +10,15 @@ export enum Side {
 }
 export type SquareWithPiece = { [key: string]: Piece };
 export type Vector = { col: number; row: number };
+export type PieceWasMoved = {
```

Comment:
Tylko nie to Types.ts :) Rozbijmy to na odrebne pliki: PieceWasMoved.ts itd. 
To nie dobrze tak dodawać do jednego - potem są konflikty rzeczy, które nawet nie zależą od siebie.

Reply:
ok, w sumie co za dużo to niezdrowo, wrzucę do osobnych plików

Reply:
Type traktujcie jak klasę / interfejs. A to zazwyczaj idzie do osobnego pliku. Te inne tez bym przeniosl, ale juz nie chcialem rodzic konfliktow - wiec to jakos przy okazji moze.